data/successor 1 introduction.txt

```diff
@@ -810,22 +814,22 @@ mission "Successors: Trusted"
 				`	"What are you doing here?"`
 					goto seiasej
 			label explain
-			`	"This is a diplomatic world for representatives of the High Houses to meet in council, resolve disputes, bicker over policy, and I suspect in as it shall be in thy own case, to offer special dispensation to citizens. I had told thou when we first met that they would be keeping their eyes upon thee, and since they have summoned thou here today, they must clearly want to involve thee in the next stage of their plans."`
+			`	"This is a diplomatic world for representatives of the High Houses to meet in council, resolve disputes, bicker over policy, and I suspect in as it shall be in thy own case, to offer special dispensation to citizens. I had told thou when we first met that they would be keeping their eyes upon thee, and since they have summoned thou here today, they most clearly want to involve thee in the next stage of their plans."`
```

Comment:
```suggestion
			`	"This is a diplomatic world for representatives of the High Houses to meet in council, resolve disputes, bicker over policy, and I suspect in as it shall be in thy own case, to offer special dispensation to citizens. I had told thou when we first met that they would be keeping their eyes upon thee, and since they have summoned thou here today, they must clearly want to involve thee in the next stage of their plans."`
```